spec/peep_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'peep'
+
+describe Peep do
+
+  describe '.new' do
+    it 'creates a new peep' do
+      Peep.new(peep: 'ever danced with the devil in the pale moonlight?')
+
+      expect(Peep.all).to include 'ever danced with the devil in the pale moonlight?'
+    end
+  end
+
+  describe '.all' do
+    it 'returns all peeps' do
+
+      add_row_to_test_database
+      
+      peeps = Peep.all
+
```

Comment:
Why are there two Peeps in the DB when you only called add_row_to_test_database once?